bubblewrap.c

```diff
@@ -453,6 +471,19 @@ drop_caps (bool all)
   if (!is_privileged)
     return;
 
+  if (opt_unshare_user)
+    {
+      caps0 |= CAP_TO_MASK (CAP_CHOWN) | CAP_TO_MASK(CAP_SETUID) | CAP_TO_MASK (CAP_SETGID) | CAP_TO_MASK (CAP_FOWNER) | CAP_TO_MASK (CAP_DAC_OVERRIDE);
+    }
```

Comment:
thanks for the rewiew.  Yes, this patch series is risky (and some parts too hacky), so I wanted to share my work in progress so to get comments before I spend more time on it.

IIUIC, even with DAC_OVERRIDE, the container should not be able to access files owned by users that have no mapping in the container user namespace.  The only users that get a mapping in the container namespace are the uid of the user using bwrap and the range 10000-10999.